hawkmoth/__init__.py

```diff
@@ -10,7 +10,7 @@
 import glob
```

Comment:
Penultimate sentence in the commit looks a bit funny: as ..., so ...
Otherwise this looks like a good set of changes too.